### PR TITLE
feat(966): add banner ui

### PR DESCRIPTION
**Pull request by russdeluca**
## Related links
Related to https://github.com/screwdriver-cd/screwdriver/issues/966

### CODE REVIEW

File: app/components/app-header/template.hbs [hunk 1]
```diff
@@ -117,6 +117,7 @@
     {{/navbar.nav}}
   {{/navbar.content}}
 {{/bs-navbar}}
+
```
[the remark this answers is not in the document]
Comment (russdeluca) [reply app/components/app-header/template.hbs]: done

File: app/application/template.hbs [hunk 2]
```diff
@@ -1,12 +1,13 @@
 {{app-header
   currentUrl=currentUrl
-  scmContexts=scmContexts
+  scmContexts=model
```
[the remark this answers is not in the document]
Comment (russdeluca) [reply app/application/template.hbs]: done

File: app/components/nav-banner/component.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import { set } from '@ember/object';
+import { inject as service } from '@ember/service';
+import Component from '@ember/component';
+
+export default Component.extend({
```
[the remark this answers is not in the document]
Comment (russdeluca) [reply app/components/nav-banner/component.js]: done

File: app/components/nav-banner/component.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  banner: service('banner'),
+
+  actions: {
+    clearMessage() {
+      this.set('message', null);
+    }
+  },
+
+  myFetchBanners() {
```
Comment (tkyi) [inline app/components/nav-banner/component.js]: Is `myFetchBanners` the best name for this method? Maybe `setBanners`?

**Comment by tkyi** [comment]
Can you add screenshots of the new banners to the pull request description?

File: app/components/nav-banner/component.js (hunk 4 above)
Comment (russdeluca) [reply app/components/nav-banner/component.js]: ahh, looking more closely, I see why you suggested setBanners.  Making that change :)

File: tests/unit/nav-banner/service-test.js [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    });
+  }
+});
+
+moduleFor('service:nav-banner', 'Integration | Component | nav banner', {
+  integration: true,
+
+  beforeEach() {
+    server = new Pretender();
+    this.register('service:nav-banner', mockBannerService);
```
Comment (jithin1987) [inline tests/unit/nav-banner/service-test.js]: Whole point is testing the banner service, this should not be mocked.

File: tests/unit/nav-banner/service-test.js [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+      'Content-Type': 'application/json'
+    },
+    '[]'
+  ]);
+};
+
+const mockBannerService = Service.extend({
+  fetchBanners() {
+    return resolve({
+      banners: [
```
Comment (jithin1987) [inline tests/unit/nav-banner/service-test.js]: return this data in `getBanners`

**Comment by russdeluca** [comment]
screwdriver-banner-info
screwdriver-banner-warn

File: app/styles/screwdriver-colors.scss [hunk 7]
```diff
@@ -5,6 +5,11 @@ $sd-white: #FFF;
 $sd-gradient-start: $sd-light-blue;
 $sd-gradient-end: $sd-dark-blue;
 
+$sd-warn-fg: #FAFBFC;
```
Comment (tkyi) [inline app/styles/screwdriver-colors.scss]: Is there any reason why this can't just be `$sd-failure`?

**Pull request merged**